src/ldp/alg/rollout.py

```diff
@@ -260,6 +343,9 @@ async def store_step(step: Transition):
         except CaughtError as e:
             # NOTE: This trajectory should not be used for regular training.
             # We save the last transition here for debugging, etc.
+            if log_exceptions_immediately:
+                logger.exception(f"Exception in rollout {traj_id}: {e.original_exc}")
+
```

Comment:
Can you bring back `format_error_details`? It's needed for remote agents/envs that communicate over HTTP (i.e. the exception is the response body)